repose-aggregator/external/service-clients/auth/src/main/java/org/openrepose/common/auth/openstack/AuthenticationService.java

```diff
@@ -12,11 +13,11 @@
  */
 public interface AuthenticationService {
 
-    AuthenticateResponse validateToken(String tenant, String userToken);
+    AuthenticateResponse validateToken(String tenant, String userToken) throws AkkaServiceClientException;
```

Comment:
Not sure how i missed this before, but the same deal as the other one, interfaces should be throwing other modules exceptions as a general rule. If they need to throw an exception, it should be one they define.